app/js/index.js

```diff
@@ -26,6 +26,7 @@
 	program
 		.version(pkg.version)
 		.option('--minimize-to-tray', 'When the tray icon is clicked, hide the window rather than minimize')
+		.option('--close-to-tray', 'When the close button is clicked ,inimize the app to tray instead of killing it.')
```

Comment:
There seems to be a typo here. I believe you meant:

"When the close button is clicked, minimize the app to tray instead of killing it."